app/src/main/java/org/mozilla/focus/fragment/HomeFragment.kt

```diff
@@ -103,8 +103,8 @@ class HomeFragment : Fragment() {
 
     private fun initTiles() = with (tileContainer) {
         val homeTiles = mutableListOf<HomeTile>().apply {
-            addAll(CustomTilesManager.getInstance(context).getCustomHomeTilesList())
             addAll(BundledTilesManager.getInstance(context).getBundledHomeTilesList())
+            addAll(CustomTilesManager.getInstance(context).getCustomHomeTilesList())
```

Comment:
The list gets reversed in `HomeTilesManager` so you should remove that line too, and return a copy.